stable/database/README.md

```diff
@@ -228,15 +228,20 @@ The following tables list the configurable parameters of the `database` chart an
 | `sm.hotCopy.successHistory` | Number of successful Jobs to keep | `5` |
 | `sm.hotCopy.failureHistory` | Number of failed jobs to keep | `5` |
 | `sm.hotCopy.backupDir` | Directory path where backupsets will be stored | `/var/opt/nuodb/backup` |
-| `sm.hotCopy.backupGroup` | Name of the backup group | `{{ .Values.cloud.cluster.name }}` |
-| `sm.hotCopy.fullSchedule` | cron schedule for FULL hotcopy jobs | `35 22 * * 6` |
-| `sm.hotCopy.incrementalSchedule` | cron schedule for INCREMENTAL hotcopy jobs | `35 22 * * 0-5` |
+| `sm.hotCopy.backupGroupPrefix` | Prefix for the automatically generated backup groups | `{{ .Values.cloud.cluster.name }}` |
+| `sm.hotCopy.backupGroups` | Backup groups configuration. By default a backup group per HCSM is created automatically | `{}` |
+| `sm.hotCopy.backupGroups.<name>.labels` | Space separated process labels used to select the Storage Managers which are part of this backup group. _Any_ label key and value should match for the SM to be selected | `nil` |
+| `sm.hotCopy.backupGroups.<name>.overwrites.fullSchedule` | cron schedule for _FULL_ hot copy performed by this backup group. If not defined the `sm.hotCopy.fullSchedule` setting will be used | `nil` |
```

Comment:
Can you replace `overwrites` with `overrides`? Or what about just `sm.hotCopy.backupGroups.<name>.fullSchedule`, `sm.hotCopy.backupGroups.<name>.incrementalSchedule`, etc. defining the override? I'm not sure if that introduces ambiguity, but it seems intuitive enough to me that I don't think we need to nest the value under `overwrites`/`overrides`.

Reply:
I agree with you. It would be easier if the structure is flat and remove `overwrites`.